### PR TITLE
Update delmailuser to allow mailboxes without TLD

### LINKED ISSUE #2171: [BUG] No valid mail address in remove, after making email.
# Bug Report

## Context

It's possible to use ```addmailuser daniel@test```, but not possible to use ```delmailuser daniel@test```.

### What is affected by this bug?

addmailuser and delmailuser

### When does this occur?

When forgetting the .com after a mail address.

### How do we replicate the issue?

CLI into container (I'm using this as a swarm service so not using setup.sh.)
1. ```addmailuser daniel@test```
2. ```delmailuser daniel@test```

### Actual Behavior

Addmailuser works, delmailuser fails

### Expected Behavior

*Either* addmailuser shouldn't work *or* delmailuser should work.

## Your Environment

As swarm service
- version: `v10.1.0`
- available RAM: `8GB`

### PR DESCRIPTION
**Pull request by danielvandenberg95**
# Description

Don't allow invalid email addresses in addmailuser.

Fixes #2171

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by NorseGaud** [comment]
So simple and yet so powerful 👍🏼😎😉

**Comment by casperklein** [comment]
Thank you for your contribution. You can create mailboxes without a TLD. So fixing `delmailuser` instead seems more appropriate to me.

**Comment by danielvandenberg95** [comment]
Changed. (Please squash merge when merging, to prevent commits showing up on addmailuser.)

**Review by casperklein — APPROVED** [review]

File: target/bin/delmailuser [hunk 1]
```diff
@@ -91,9 +91,6 @@ for EMAIL in "${@}"
 do
   ERROR=false
 
-  # very simple plausibility check
-  [[ ${EMAIL} != *@*.* ]] && errex "No valid email address: ${EMAIL}"
```
Comment (NorseGaud) [inline target/bin/delmailuser:95]: Could we instead mimic what addmailuser does so that there is still some validation in this file?

**Review by NorseGaud — CHANGES_REQUESTED** [review]

File: target/bin/delmailuser (hunk 1 above)
Comment (georglauterbach) [reply target/bin/delmailuser:95]: I already wanted to agree, but then, think about it: If you were able to add addresses, you should be able to remove them, and restrictions would only hinder you here. We have a "closed context" in this case, i.e. we only remove lines in one file.

I'm fine with it either way though:)

**Review by georglauterbach — APPROVED** [review]

File: target/bin/delmailuser (hunk 1 above)
Comment (danielvandenberg95) [reply target/bin/delmailuser:95]: I don't fully get the script below here, is there currently a warning if the mailbox couldn't be found / doesn't exist? If not, I think that should be added, and if so I think this suggestion is covered by that warning?

Comment (casperklein) [reply target/bin/delmailuser:95]: > is there currently a warning if the mailbox couldn't be found / doesn't exist? 

There should be a warning, however the check is currently broken. I've already fixed it locally and will provide a PR once this got merged.

Comment (georglauterbach) [reply target/bin/delmailuser:95]: @NorseGaud are you fine with this change anyway?

Comment (danielvandenberg95) [reply target/bin/delmailuser:95]: In that case I'm resolving this conversation, as that warning will inform users. I'll request a re-review from @NorseGaud.

Comment (NorseGaud) [reply target/bin/delmailuser:95]: @georglauterbach go ahead

**Comment by georglauterbach** [comment]
@casperklein You can go ahead and provide the PR you mentioned in the comments :)

**Pull request merged**